gulpfile.js

```diff
@@ -13,8 +13,7 @@ let es = require('event-stream');
 let shell = require('gulp-shell');
 let replace = require('gulp-replace');
 //let replace = require('gulp-regex-replace');
-let inject = require('gulp-file-inject');
-
+var footer = require('gulp-footer');
```

Comment:
remove 'gulp-footer' from this file and package.json since we did not use this library

Reply:
done